toolkit/tools/pkg/imagecustomizerlib/imagecustomizer_test.go

```diff
@@ -20,6 +20,8 @@ import (
 func TestCustomizeImageEmptyConfig(t *testing.T) {
 	var err error
 
+	t.Skip("Unreliable test")
```

Comment:
I think the main problem here is less about the test and more about the environment. Is there a way we could skip it conditionally? For instance if `buildpipeline.IsRegularBuild()` returns `false`?

Reply:
Good call.